testing/src/test/java/io/stargate/it/http/GraphqlTest.java

```diff
@@ -1050,8 +1050,6 @@ public void invalidTypeMappingReturnsErrorResponse() {
   private Map<String, Object> executePost(String path, String query) throws IOException {
     OkHttpClient okHttpClient = getHttpClient();
     String url = String.format("http://%s:8080%s", stargate.seedAddress(), path);
-    HttpUrl.Builder httpBuilder = HttpUrl.parse(url).newBuilder();
-    httpBuilder.addQueryParameter("query", query);
```

Comment:
See my comment in `GraphqlResourceBase.postJson`.

graphql-java-servlet was apparently more lenient when the query is both in the URI and the request body, but I think it's better to error out.